test/interpolate.test.ts

```diff
@@ -0,0 +1,183 @@
+import {interpolateSearch} from "../src/base";
+import {InterpolatorConstant} from "../src/constant";
+import {InterpolatorLinear} from "../src/linear";
+import {InterpolatorSpline, solve} from "../src/spline";
+
+// look in odin:tests/testthat/test-js-support-interpolate.R for test cases
+describe("interpolateSearch can find points", () => {
+    it("Can find point", () => {
```

Comment:
I think there's a better name for this test perhaps, since it's only testing whether you can find limit points?